localization/en/strings.json

```diff
@@ -1,4 +1,15 @@
 {
+    "add_comment": "Add a comment",
+    "friends_commenter": "Post Comments to Selected Friends",
+    "friends_commenter_warning": "Please make sure you entered a message and selected 1 or more friends.",
+    "friends_commenter_no_links": "Please remove any links from your message.",
+    "posted_at_success": "Successfully posted comment on",
+    "posted_at_fail": "Failed to post comment on",
```

Comment:
These should have substitution variables

Reply:
what do you mean?